src/app/examples/modal-example/first-embedded-modal-example/first-embedded-modal-example.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, ViewContainerRef } from '@angular/core';
+
+import { ModalController } from '~/kirby/components/modal/services/modal.controller';
+import { ModalConfig } from '~/kirby/components/modal/config/modal-config';
+import { SecondEmbeddedModalExampleComponent } from '../second-embedded-modal-example/second-embedded-modal-example.component';
+
+@Component({
+  templateUrl: './first-embedded-modal-example.component.html',
+})
+export class FirstEmbeddedModalExampleComponent {
+  constructor(private modalController: ModalController, private vcRef: ViewContainerRef) {}
+
+  async openSecondModal() {
+    const config: ModalConfig = {
+      title: 'Second Embedded Modal',
+      titleHorizontalAlignment: 'left',
+      closeIconName: 'arrow',
```

Comment:
`kirby-icon` currently has no support for `arrow`, consider using `arrow-back` for now (or else it will render a coq wheel)